samples/dashboards/demo/cloud-monitoring/demo.css

```diff
@@ -0,0 +1,144 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/datagrid/css/datagrid.css");
+@import url("https://code.highcharts.com/dashboards/css/dashboards.css");
+
+.highcharts-description {
+    padding: 0 20px;
+}
+
+.cloud-monitoring-data-controls {
+    background-color: var(--dashboard-bck-gray);
+    border-bottom: 1px solid #a5abb1;
+    padding: 20px 15px;
+}
+
+:root,
+.highcharts-light {
+    /* Colors for data series and points */
+    --highcharts-color-0: #33a29d;
+    --highcharts-color-2: #fe9d00;
+
+    /* Extra colors */
+    --bck-gray: #f1f5f9;
+    --dashboard-bck-gray: #e4eaf1;
+}
+
+body {
+    background-color: var(--bck-gray);
+}
+
+.highcharts-dashboards-wrapper {
+    background-color: var(--dashboard-bck-gray);
+    min-height: 1000px;
+}
+
+.highcharts-dashboards-cell > .highcharts-dashboards-component {
+    border-radius: 15px;
+    padding: 10px;
+    text-align: left;
+}
+
+.highcharts-dashboards-component-title {
+    font-size: 1.2em;
+    text-align: left;
+}
+
+#instance-details .highcharts-dashboards-component-content p {
+    display: inline-block;
+    font-size: 1em;
+    font-weight: 600;
+    margin-top: 20px;
+}
+
+#instance-details .highcharts-dashboards-component-content span {
+    width: 40px;
+    height: 40px;
+    background-size: 32px 32px;
+    border-radius: 5px;
+    margin: 10px 15px 10px 10px;
+    float: left;
+}
+
+/* https://awsicons.dev/ */
+
+#instance-details #instance .highcharts-dashboards-component-content span {
+    background: url("https://www.highcharts.com/samples/graphics/dashboards/cloud-monitoring/instance-ico.svg") 0 50% no-repeat;
+}
+
+#instance-details #zone .highcharts-dashboards-component-content span {
+    background: url("https://www.highcharts.com/samples/graphics/dashboards/cloud-monitoring/zone-ico.svg") 0 50% no-repeat;
+}
+
+#instance-details #ami .highcharts-dashboards-component-content span {
+    background: url("https://www.highcharts.com/samples/graphics/dashboards/cloud-monitoring/ami-ico.svg") 0 50% no-repeat;
+}
+
+#instance-details #os .highcharts-dashboards-component-content span {
+    background: url("https://www.highcharts.com/samples/graphics/dashboards/cloud-monitoring/os-ico.svg") 0 50% no-repeat;
+}
+
+#instances-table .highcharts-datagrid-container {
+    border: none;
+    font-weight: 100;
+}
+
+#instances-table .highcharts-datagrid-column-header,
+#instances-table .highcharts-datagrid-row {
+    background-color: var(--highcharts-neutral-color-0);
+}
```

Comment:
It doesn't look good in dark mode, the data grid background is completely black.